src/qt/bitcoinunits.cpp

```diff
@@ -38,11 +39,12 @@ bool BitcoinUnits::valid(int unit)
 
 QString BitcoinUnits::name(int unit)
 {
+    const QString baseUnit = QString::fromStdString(CURRENCY_UNIT);
     switch(unit)
     {
-    case BTC: return QString("BTC");
-    case mBTC: return QString("mBTC");
-    case uBTC: return QString::fromUtf8("μBTC");
+    case BTC: return QString("%1").arg(baseUnit);
```

Comment:
Actually just `baseUnit` alone would be enough.  I chose the current style because it is in my opinion more readable, since all three cases look the same.  But since this is a matter of taste, I can as well change the code if everyone thinks that I should just return `baseUnit` here.